src/main/java/online/book/store/service/book/BookServiceImpl.java

```diff
@@ -28,14 +29,14 @@ public BookResponseDto save(BookRequestDto requestDto) {
     }
 
     @Override
-    public List<BookResponseDto> findAll(Pageable pageable) {
+    public List<BookResponseDto> getAll(Pageable pageable) {
```

Comment:
You should also modify save method to ensure that categories that you are trying to add to book exist